build/Targets/GenerateInternalsVisibleTo.targets

```diff
@@ -1,3 +1,4 @@
+<?xml version="1.0" encoding="Windows-1252"?>
```

Comment:
Had a similar thing happen before.  I'm not sure what tool adds this but guessing some edit experience in VS does it.  It's weird.